lib/puppet_x/sqlserver/sql_connection.rb

```diff
@@ -90,25 +90,26 @@ def win32_exception
     end
 
     class ResultOutput
+
       attr_reader :exitstatus, :error_message, :raw_error_message
```

Comment:
So this class still has a `raw_error_message` attribute... but it's not set anymore? Do we still want to keep the concept of what this previously represented?

Reply:
No, I'll get rid of it. That was previously in place because the larger error message was being parsed for the sql error, but even in the older version it was never called.